### ISSUE TITLE
We need new sustaining member icons for QGIS

### ISSUE THREAD

**Issue by timlinux** [kartoza/prj.app#1103]
@anitagraser Can you come up with something a bit fresher looking?

CC @andreasneumann

![Screenshot from 2020-01-09 10-28-49](https://user-images.githubusercontent.com/178003/72060003-0614d580-32cb-11ea-849a-674fcb7fc310.png)

**Comment by anitagraser** [comment]
We already have new graphics in the QGIS Graphics directory on Google Drive. They look like this:

![qgis-bronze-sponsor-years](https://user-images.githubusercontent.com/590385/72064937-4bda9980-32dd-11ea-9c8c-22f5ace3b28f.png)

I see that Andreas already started reworking them to replace "sponsor" with "sustaining member"

**Comment by timlinux** [comment]
Nice @anitagraser can we some how get that into a square format?

**Comment by anitagraser** [comment]
I'll see what I can do. Do you want dates (years) in the graphics or just "QGIS small/medium/large sustaining member"?

**Comment by andreasneumann** [comment]
I think there would be use cases for both: with and without date. But definitely, we should have some without dates to replace these sponsor badges.
Thanks Anita!

**Comment by anitagraser** [comment]
What do you think of: 

![small_member](https://user-images.githubusercontent.com/590385/73135433-1acec880-4042-11ea-929a-af15899b2d1c.png)

**Comment by andreasneumann** [comment]
Looks good to me. Would you vary the colors according to membership size or not? I don't have a strong opinion either way. Thanks, Andreas.

**Comment by anitagraser** [comment]
Yes, I'd stick to the bronze / silver / gold colors we had, even though the words are not used anymore.

**Comment by anitagraser** [comment]
So the other two would be: 
![medium_member](https://user-images.githubusercontent.com/590385/73139668-9a738c00-4070-11ea-9c17-a562802e86b7.png)
![large_member](https://user-images.githubusercontent.com/590385/73139671-9c3d4f80-4070-11ea-8306-e62836dab54e.png)

**Comment by timlinux** [comment]
I wonder how this will look when scaled down to a thumbnail Anita?

**Comment by anitagraser** [comment]
@timlinux do you mean the size of the images in your initial post?

**Comment by anitagraser** [comment]
![72060003-0614d580-32cb-11ea-849a-674fcb7fc310](https://user-images.githubusercontent.com/590385/73158500-46a28a80-40e4-11ea-8da1-c0f0a8c3af6c.png)

**Comment by timlinux** [comment]
Hi @anitagraser 

So I published your new images:

![image](https://user-images.githubusercontent.com/178003/74830263-a4547c00-530a-11ea-9df8-3f1938366091.png)


Could you make a 'Flagship' one to replace the platinum one we had? I have used Large for now. 

One thing I want to suggest is having the colours go from bright for high sponsorship levels to dull for small ones. At the moment the small level catches my eye more than the high ones do and I thinkg it should be the opposite.

https://changelog.qgis.org/en/qgis/membershiplevel/list/ <-- listing shown here

![image](https://user-images.githubusercontent.com/178003/74830429-08774000-530b-11ea-82b5-f234018cbc1f.png)


I think we should also advise our sponsors to provide a square formatted logo and tell them the different sizes it will be used at so they can put together nice looking logos for these list pages.

**Comment by timlinux** [comment]
sponsors -> sustaining members above

**Comment by anitagraser** [comment]
@timlinux Looking at the logo uses in your examples, it would seem that we wouldn't need to stick to a square design but could instead make wider logos or am I missing something?

**Comment by timlinux** [comment]
> @timlinux Looking at the logo uses in your examples, it would seem that we wouldn't need to stick to a square design but could instead make wider logos or am I missing something?

Yes we could use a rectangular format but I think we should still establish guidelines so that sustaining members can upload their logos knowing they will be shown nicely in the various places.

**Comment by anitagraser** [comment]
@timlinux sorry for being unclear, I meant that our QGIS badges don't need to be square

**Comment by timlinux** [comment]
> @timlinux sorry for being unclear, I meant that our QGIS badges don't need to be square

Agreed they could be another shape  (though I do quite like the square shape)

**Comment by anitagraser** [comment]
> One thing I want to suggest is having the colours go from bright for high sponsorship levels to dull for small ones. 

I've implemented this idea now:

![flagship_member](https://user-images.githubusercontent.com/590385/76936078-8fabd980-68f2-11ea-958d-8a30911d37f3.png)
![large_member](https://user-images.githubusercontent.com/590385/76936116-a05c4f80-68f2-11ea-9622-5b3d6cfcccc9.png)
![medium_member](https://user-images.githubusercontent.com/590385/76936082-90dd0680-68f2-11ea-9202-38c8a833575e.png)
![small_member](https://user-images.githubusercontent.com/590385/76936083-91759d00-68f2-11ea-9dec-3b34b410b302.png)

**Comment by timlinux** [comment]
Awesome, thanks so much!
